<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ClassifyDialogBase</class>
 <widget class="QDialog" name="ClassifyDialogBase">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Unsupervised Classifier</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QComboBox" name="algorithmComboBox"/>
   </item>
   <item>
    <widget class="QSpinBox" name="numClustersSpinBox">
     <property name="minimum">
      <number>2</number>
     </property>
     <property name="maximum">
      <number>10</number>
     </property>
     <property name="value">
      <number>5</number>
     </property>
     <property name="prefix">
      <string>Number of Clusters: </string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="isodataOptionsGroupBox">
     <property name="title">
      <string>ISODATA Options</string>
     </property>
     <layout class="QFormLayout" name="formLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Max Iterations</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QSpinBox" name="maxIterSpinBox">
        <property name="maximum">
         <number>1000</number>
        </property>
        <property name="value">
         <number>100</number>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Max Merge</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QDoubleSpinBox" name="maxMergeDoubleSpinBox">
        <property name="maximum">
         <number>10.0</number>
        </property>
        <property name="value">
         <double>0.5</double>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Min Split Std</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QDoubleSpinBox" name="minSplitStdDoubleSpinBox">
        <property name="maximum">
         <number>10.0</number>
        </property>
        <property name="value">
         <double>0.5</double>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>Max Std</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QDoubleSpinBox" name="maxStdDoubleSpinBox">
        <property name="maximum">
         <number>10.0</number>
        </property>
        <property name="value">
         <double>1.0</double>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>Min Samples</string>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QSpinBox" name="minSamplesSpinBox">
        <property name="maximum">
         <number>1000</number>
        </property>
        <property name="value">
         <number>10</number>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QSpinBox" name="numBandsSpinBox">
     <property name="minimum">
      <number>1</number>
     </property>
     <property name="maximum">
      <number>10</number>
     </property>
     <property name="value">
      <number>4</number>
     </property>
     <property name="prefix">
      <string>Number of Bands: </string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QListWidget" name="selectedBandsListWidget"/>
   </item>
   <item>
    <widget class="QPushButton" name="runButton">
     <property name="text">
      <string>Run Clustering</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
